packages/experience-builder-sdk/src/utils/prebindingUtils.spec.ts

```diff
@@ -294,8 +294,10 @@ describe('resolveMaybePrebindingDefaultValuePath', () => {
             ...experienceEntry.fields.componentSettings,
             patternPropertyDefinitions: {
               testPatternPropertyDefinitionId: {
-                defaultValue: {
-                  testContentType: {
+                defaultSourcs: {
```

Comment:
```suggestion
                defaultSource: {
```
This breaks the unit test right now